.changeset/techdocs-swift-waves-sleep.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-techdocs': patch
```

Comment:
The existing types are exported, so expecting that this would need to be `minor`